### PR TITLE
Accessability fixes for vcpkg home and package pages

### PR DESCRIPTION
**Pull request by Sreenivas-Kalluru**
The fix for the following accessibility bugs

- https://netsites.visualstudio.com/msc/_workitems/edit/9101/
- https://netsites.visualstudio.com/msc/_workitems/edit/9102/
- https://netsites.visualstudio.com/msc/_workitems/edit/9103

### CODE REVIEW

File: en/index.html [hunk 1]
```diff
@@ -265,7 +265,7 @@
     <div class="why-us-box-shadow">
       <div class="why-us-flexbox">
         <div class="why-us-group">
-          <img class="why-us-icon" src="/assets/mark/easy.svg" />
+          <img class="why-us-icon" src="/assets/mark/easy.svg" aria-label="easy icon" />
```
Comment (ras0219-msft) [inline en/index.html:268]: From MDN[1]:
> ... aria-label is intended for use on interactive elements ...

These images aren't interactive (if I understand correctly). Do we want alt text here instead? Or maybe should these be aria-labelledby the following headings?

[1]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

Comment (Sreenivas-Kalluru) [reply en/index.html]: @ras0219-msft, I have incorporated review comments. Could you please review and trigger the build on these changes.

**Review by Sreenivas-Kalluru — COMMENTED** [review]
@ras0219-msft  Could you please review

**Comment by ras0219-msft** [comment]
I don't think you addressed my comment:

> Should aria-label be human understandable or should it be based on the filename?
>
> It looks like our icon is misnamed -- the "Reliable" section is below.

Specifically, https://vcpkg.io/assets/mark/reliable.svg ended up being used to describe "Universal".

File: en/index.html [hunk 2]
```diff
@@ -275,7 +275,7 @@
           </p>
         </div>
         <div class="why-us-group">
-          <img class="why-us-icon" src="/assets/mark/reliable.svg" />
+          <img class="why-us-icon" src="/assets/mark/reliable.svg" aria-label=" reliable icon"  />
```
[the remark this answers is not in the document]
Comment (Sreenivas-Kalluru) [reply en/index.html]: @ras0219-msft  alt text updated, Could you please review and deploy this changeset

**Review by Sreenivas-Kalluru — COMMENTED** [review]
@ras0219-msft Could you please review

**Review by ras0219-msft — APPROVED** [review]

**Review by Sreenivas-Kalluru — COMMENTED** [review]
@ras0219-msft  I have made changes for the nav bar template, Could you please review and merge the changes.

**Comment by ras0219-msft** [comment]
There are still validation issues.

You need to edit the templates appropriately, then run `rebuild.sh` to regenerate the `/en/` folder.

**Review by Sreenivas-Kalluru — COMMENTED** [review]
@ras0219-msft , I have regenerated the website with template updates. Could you please review and merge the changes.

**Comment by ras0219-msft** [comment]
I have fixed the validation issues in a branch, please pull it:

```
git pull https://github.com/ras0219-msft/vcpkg.github.io sreenivas-kalluru
```

**Pull request merged**